aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -800,4 +801,113 @@ Tensor sspaddmm(const Tensor& self, const Tensor& mat1, const Tensor& mat2,
   return result;
 }
 
+// --------------------------------------------------------------------
+// sparse_sum
+// --------------------------------------------------------------------
+Tensor sparse_sum(const SparseTensor& t) {
+  return at::sparse_sum(t, t.dtype());
+}
+
+Tensor sparse_sum(const SparseTensor& t, ScalarType dtype) {
+  return t._values().sum().to(dtype);
```

Comment:
If I say `sparse_sum(float_tensor, kDouble)`, do I want the accumulation to be done in double precision? Seems likely to me. But that's not what this code does.

Reply:
Yeah, so the behavior here is inconsistent with `torch.sum`